tests/js-modules/modules.py

```diff
@@ -1106,7 +1106,54 @@ def corrupt_value(value: str):
+JukpgB1c3yBP1Mm5XkZzawBPoAEmlTgz7QnDnz1RbE36beNIHKLddBtkdNMoPxmg1ztlJdPr0PENmF5Sdm+CGO7YIUORlwiOQ2YKlV+/V6Gk7As4w8+anGlo2HqqeTGBes35yRmG+zsJ+5gigrBwZLepvjh2zdE4a195Kl4Mp4SGqpEJLZSFKIYeGw3DLiSjkGgbhXmyQhqUQIN8uIc5xKA8JGTmQQ7KzHl7Dto8eeHzvoqemlwT+QUD0o1C9P4w3jCramzAZK35fvm3bHuOIUKuplI8EjZablf8RE713l7AOrlTKcd1KT8uaqWgzmuxMmtZDnw05I7CXIjcZHmZhVr7zbOSBV7ux5E7CLZ6+E2AgNKQ8iSttsfSZOriLxAT+RGxeqz9w/NgaXQAVTn4IL2aKh9lFfEYvIo2b+46PJ8Knu6MMQ==
+""",
+            }
+
+            r = c.post(
+                "/app/verifySnpAttestation",
+                {
+                    "evidence": reference_quote["raw"],
+                    "endorsements": reference_quote["endorsements"],
+                    "uvm_endorsements": reference_quote["uvm_endorsements"],
+                    "endorsed_tcb": reference_quote["endorsed_tcb"],
+                },
+            )
+            assert r.status_code == http.HTTPStatus.OK, r.status_code
+            assert r.body.json()["attestation"]["report_data"].startswith(
+                "949bfd58322a767aeb4e0093eab102b36eb90e63e9398a7d471347efebd4f0fb"
```

Comment:
Why startswith and not equality?

Reply:
I believe that the report data for CCF is the SHA256 hash of a nodes public key, which is 32 bytes so only half of the available 64 bytes in report data, so the remaining 32 bits are zeros.

Thinking about it though, I reckon it's probably safer to assert that exactly